bin/web.js

```diff
@@ -40,6 +40,8 @@ function middleware (id, meta, body, respond, req, res, resource, verb, endpoint
   // function called before endpoints
   // authentications and other prerequisites when necessary
   if (typeof req.headers['x-real-ip'] === 'string') {
+    // get real ip
+    const browser_ip = req.headers['x-real-ip'] 
```

Comment:
```suggestion
    const browserIp = req.headers['x-real-ip'] 
```

Cammel case https://standardjs.com/